packages/neuron-wallet/src/controllers/multisig.ts

```diff
@@ -2,15 +2,40 @@ import fs from 'fs'
 import path from 'path'
 import { dialog, BrowserWindow } from 'electron'
 import { t } from 'i18next'
-import { addressToScript, scriptToHash } from '@nervosnetwork/ckb-sdk-utils'
+import { addressToScript, scriptToAddress, scriptToHash } from '@nervosnetwork/ckb-sdk-utils'
 import { ResponseCode } from 'utils/const'
 import MultisigConfig from 'database/chain/entities/multisig-config'
 import MultisigConfigModel from 'models/multisig-config'
 import MultisigService from 'services/multisig'
 import CellsService from 'services/cells'
 import OfflineSignService from 'services/offline-sign'
+import Multisig from 'models/multisig'
+import SystemScriptInfo from 'models/system-script-info'
+import NetworksService from 'services/networks'
+
+interface MultisigConfigOutput {
+  multisig_configs: Record<string, {
+    sighash_addresses: string[],
+    require_first_n: number,
+    threshold: number
+    alias?: string
+  }>
+}
+
+const validImportConfig = (configOutput: MultisigConfigOutput) => {
```

Comment:
`validImportConfig` => `validateImportConfig` or `isImportConfigValid`

Reply:
Add tests on this method

Reply:
I have tested it by `importConfig` function.